### ISSUE TITLE
MathJax in preview

### ISSUE THREAD

**Issue by dthies** [mudrd8mz/moodle-editor_marklar#10]
Hello David, 
Very useful project. I noticed that MathJax is not processing in the preview. I believe it is working with TeX Notation filter. To fix this MathJax needs to be lloaded on the page, and a filter_content_update event should be fired by AJAX.
Daniel

**Comment by mudrd8mz** [comment]
Thanks Daniel. Maybe the Moodle Fragments API could be used here to catch such cases in general. I must admit I would prefer not to hard-code an exceptional behaviour for the MathJax. But if there is no other easy way, I would not mind it. I fully understand the value of previewing math filters.

**Comment by dthies** [comment]
Yes, it looks like this is one of the issues that the Fragments API  is intended to address.

**Comment by mudrd8mz** [comment]
Is this still an issue? 0.8.2 should have it fixed.
